### PR TITLE
RTL implementation for the HTML5 client

### PR DESCRIPTION
**Pull request by GhaziTriki**
Work in progress.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/app/component.jsx [hunk 1]
```diff
@@ -258,7 +258,7 @@ class App extends Component {
         <PollingContainer />
         <ModalContainer />
         <AudioContainer />
-        <ToastContainer />
+        <ToastContainer rtl />
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/app/component.jsx]: Why is `rtl` needed here for this container? Isn't `document.body.parentNode.setAttribute('dir', 'rtl');` sufficient?

File: bigbluebutton-html5/imports/startup/client/intl.jsx [hunk 2]
```diff
@@ -111,19 +113,29 @@ class IntlStartup extends Component {
...
-            Settings.save();
+            this.saveLocale(DEFAULT_LANGUAGE);
           });
         });
     });
   }
 
+  saveLocale(localeName) {
+    Settings.application.locale = localeName;
+    if (RTL_LANGUAGES.includes(localeName)) {
```
Comment (antobinary) [inline bigbluebutton-html5/imports/startup/client/intl.jsx]: Is anything extra needed to support a future `ar_TN` locale? I see that `const RTL_LANGUAGES = ['ar', 'he', 'fa'];` are only the first 2 letters, so `localeName` might be `ar_TN` while `RTL_LANGUAGES` is just `'ar'`

File: bigbluebutton-html5/imports/ui/components/app/component.jsx (hunk 1 above)
Comment (GhaziTriki) [reply bigbluebutton-html5/imports/ui/components/app/component.jsx]: I just followed the docs https://github.com/fkhadra/react-toastify#right-to-left-support

**Comment by ffdixon** [comment]
Yay!

**Comment by ffdixon** [comment]
Fixes #1441

**Comment by Jossef-767** [comment]
when multi user whiteboard is on, the name label on whiteboard is choppy or missing

**Comment by capilkey** [comment]
I don't see any choppiness, but the names definitely aren't rendering correctly in RTL.

image

**Comment by Jossef-767** [comment]
The same problem also in poll results
image

**Pull request merged**